src/hooks/update.ts

```diff
@@ -1,12 +1,13 @@
 import {Hook} from '@oclif/core'
 
-import Plugins from '../plugins'
+import Plugins from '../plugins.js'
 
 export const update: Hook<'update'> = async function () {
   const plugins = new Plugins(this.config)
   try {
     await plugins.update()
-  } catch (error: any) {
-    this.error(error.message)
+  } catch (error: unknown) {
+    const {message} = error as {message: string}
```

Comment:
this trusts that nothing down the stack throws anything that doesn't match the js Error class.  

conditional would be better (ex: `if (error instanceof Error)` or `if ('message' in error)` etc.

this.error doesn't handle undefineds (according to its types)